### ISSUE TITLE
Archive inms.csv appears incomplete

### ISSUE THREAD

**Issue by alex-evans** [red-4/curious-moon#61]
I'm working through the inms.csv file import steps in the Sniff the Sky section and hitting issues with the data. I'm assuming it's an issue on my end but the inms.csv file I pulled from the Archives for the Cassini Mission appears to be incomplete. The final timestamp in my version is "2008-283T18:49:32.257" and it's 1.31 GBs. 

I'll pull from the raw data but wanted to first make sure I wasn't doing anything dumb or if it actually is not the complete dataset in the archive. I was assuming it would be the csv file Dee eventually gets created after the USING CSVSQL chapter and uses in the SUCCESSFUL IMPORT chapter (pg 205 of the pdf I'm working with).

Thanks. Great book.

**Comment by rothjm** [comment]
I also had an issue extracting inms.csv

**Comment by robconery** [comment]
I'm wondering if this is related? https://github.com/red-4/curious-moon/issues/23

**Comment by rothjm** [comment]
Yes!  My bad, thanks for the assistance!

**Comment by robconery** [comment]
No problem - @alex-evans did you ever get this sorted?

**Comment by alex-evans** [comment]
Nah, I ended up just using the raw data. Seems like I'm the only one that had this problem so probably just a me problem. I'm on a Mac so don't think it's related to issue #23 . Thanks though, I didn't hit any other problems and enjoyed the book a lot. This can be closed out probably.

**Comment by claytron** [comment]
I ran into the same issue when unarchiving via Finder on macOS, then trying to import the data, getting the following error:

```
psql:/Users/clayton/curious_moon/scripts/import_inms.sql:81: ERROR:  missing data for column "lst_s"      
CONTEXT:  COPY inms, line 3339530: "2008-283T18:49:32.257,67772257,"ENCELADUS",-1027467,56216,232441,-23,"csn",0,"199-1"
,1,"High-EV","Of..."
```

But was able to successfully do it from the command line with the stock `unzip`:

```sh
$ unzip cassini_data.zip
$ ls -lh curious_data/data/INMS/inms.csv 
-rwxr-xr-x@ 1 clayton  staff   5.2G Nov 20  2017 curious_data/data/INMS/inms.csv
```

**Comment by robconery** [comment]
Yep - considering closing this issue as 7-zip and command line do a much better job with large files. There's nothing I can do about that.
